system/views/formcontrols/readonly/index.cfm

```diff
@@ -1 +1,11 @@
-<cfoutput><span class="read-only">#( HTMLEditFormat( args.defaultValue ?: ( args.savedValue ?: "" ) ) )#</span></cfoutput>
\ No newline at end of file
+<cfscript>
+	renderer = args.renderer     ?: "";
+	context  = args.context      ?: "default";
```

Comment:
use "readonly" context + make `args.rendererContext` to avoid conflicting "contexts" (forms can be rendered in a context)